docs/docs/architecture/components/coordinator.md

```diff
@@ -18,12 +18,27 @@ The manifest is the configuration file for the Coordinator, defining your confid
 It's automatically generated from your deployment by the Contrast CLI.
 It currently consists of the following parts:
 
+<!-- TODO(burgerdev): explain manifest on separate page. -->
+
 - _Policies_: The identities of your Pods, represented by the hashes of their respective runtime policies.
 - _Reference Values_: The remote attestation reference values for the Kata confidential micro-VM that's the runtime environment of your Pods.
 - _WorkloadOwnerKeyDigest_: The workload owner's public key digest. Used for authenticating subsequent manifest updates.
 - _SeedshareOwnerKeys_: public keys of seed share owners. Used to authenticate user recovery and permission to handle the secret seed.
 
-<!-- TODO(burgerdev): document manifest storage. -->
+## Manifest history
+
+The Coordinator uses Kubernetes `ConfigMap`s to store the manifest history and associated runtime policies.
+The manifest isn't considered sensitive information, because it needs to be passed to the untrusted infrastructure in order to start workloads.
```

Comment:
```suggestion
Manifest and policies aren't considered sensitive information, because it needs to be passed to the untrusted infrastructure in order to start workloads.
```